File: cli_ver.php
<?php
include('funciones.php');
include('datos.php');
if (!$_GET['accion']){
    $accion='listado';
}
else{
    $accion=$_GET['accion'];
}
/* Recuperar variables */
$cliente_id = $_GET['id'];
encabezado('Detalles del cliente');
$fil1 = buscar_registro("clientes","id",$cliente_id);
encabezado_tabla(array("Cliente:",$fil1['nombre']));
echo '<tr>';
celda("Dirección:"); celda($fil1["direccion"]);
echo '</tr>';
echo'<tr>';
celda("Teléfono:"); celda($fil1["telefono"]);
echo '</tr>';
echo '<tr>';
celda("eMail:"); celda($fil1["email"]);
echo '</tr>';
$sql_catcli = 'SELECT * FROM cat_clientes WHERE id="'.$fil1[7].'"';
$res_catcli = mysql_query($sql_catcli);
$fil_catcli = mysql_fetch_row($res_catcli);
$categoria = $fil_catcli[1];
echo '<tr>';
celda("Categoría:"); celda($categoria);
echo '</tr>';
echo '<tr>';
celda("CI:"); celda($fil1["ci"]);
echo '</tr>';
echo '<tr>';
celda("Notas:"); celda($fil1["notas"]);
echo '</tr>';
$sql_alu = "SELECT * FROM alumnos WHERE cliente_id = '$cliente_id'";
$res_alu = mysql_query($sql_alu);
while($fil_alu = mysql_fetch_array($res_alu)){
    echo '<tr>';
    $fil_cur = buscar_registro("cursos","id",$fil_alu['curso_id']);
    if($fil_alu['finalizado']==1){
        celda("Cursó:");
    } else {
        celda("Inscripto en:");
    }
    celda($fil_cur['curso']);
    echo '</tr>';
}
fin_tabla();
boton("Estado de cuenta",'ctacli.php?id='.$fil1['id'].'&accion=listado');
boton("Llamadas",'lla.php?accion=listado&cliente_id='.$_GET['id']);
boton("Notas",'cli_ver.php?id='.$_GET['id'].'&accion=notas');
boton("Escolaridad",'cli_ver.php?id='.$_GET['id'].'&accion=escolaridad');
boton("Documentos",'cli_ver.php?id='.$_GET['id'].'&accion=documentos');
boton("Volver","cli.py?accion=listado");
?>

File: cli_ver.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
import cgi 
import cgitb; cgitb.enable()
import datos
import htm
import pagina
def listado(frm):
    """Listado de detalles del cliente"""
    # Recuperar variables
    cliente_id = frm.getvalue("id")
    # Bases de datos
    clientes = datos.Tabla("clientes")
    cat_clientes = datos.Tabla("cat_clientes")
    alumnos = datos.Tabla("alumnos")
    cursos = datos.Tabla("cursos")
    # Busquedas
    clientes.ir_a(cliente_id)
    cat_clientes.ir_a(clientes.registro["categoria_id"])
    alumnos.filtro = "cliente_id=" + str(cliente_id)
    alumnos.filtrar()
    # Pagina
    pag = pagina.Pagina('Detalles del cliente', 5)   
    htm.encabezado_tabla(['Cliente:', clientes.registro["nombre"]])
    print(htm.tr(htm.td('Dirección:') + htm.td(clientes.registro["direccion"])) +
        htm.tr(htm.td("Teléfono:") + htm.td(clientes.registro["telefono"])) +
        htm.tr(htm.td("eMail:") + htm.td(clientes.registro["email"])) +
        htm.tr(htm.td("Categoría:") + htm.td(cat_clientes.registro["categoria"])) +
        htm.tr(htm.td("CI:") + htm.td(clientes.registro["ci"])) +
        htm.tr(htm.td("Notas:") + htm.td(clientes.registro["notas"])))
    htm.fin_tabla()
    htm.encabezado_tabla(["Estado", "Curso"])
    i = 0
    for fila in alumnos.resultado:
        cursos.ir_a(fila["curso_id"])
        htm.fila_alterna(i)
        if cursos.registro["finalizado"] == 1:
            print(htm.td("Cursó:"))
        else:
            print(htm.td("Inscripto en:"))
        print(htm.td(cursos.registro["curso"]))
        print('</tr>')
        i = i + 1
    htm.fin_tabla()
    print(htm.button('Estado de cuenta', "ctacli.php?id=" + str(cliente_id) + "&accion=listado"))
    print(htm.button('Llamadas', "lla.php?accion=listado&cliente_id=" + str(cliente_id)))
    # print(htm.button('Notas', "cli_ver.py?id=" + str(cliente_id) + "&accion=notas"))
    # print(htm.button('Escolaridad', "cli_ver.py?id=" + str(cliente_id) + "&accion=escolaridad"))
    # print(htm.button('Documentos', "cli_ver.py?id=" + str(cliente_id) + "&accion=documentos"))
    print(htm.button('Volver', 'cli.py?accion=listado'))
    pag.fin()
if __name__ == "__main__":
    form = cgi.FieldStorage()
    accion = form.getvalue("accion", "listado")
    if accion == "listado":
        listado(form)
    else:
        listado(form)